<launch>


    <node pkg="steel_detection" type="steel_detect" name="steel_detect"/>



</launch>